Analysis/trigger.tex

```diff
@@ -1,10 +1,10 @@
 We measure the trigger efficiency by looking for trigger objects that match the candidate photon object in an appropriate data set because the trigger decisions are based on the existence of a single photon object in the event.
 A trigger object is the four-momenta of an object reconstructed at the trigger level that is used for making trigger decisions. 
-A trigger object is matched to the candidate when their angular separation  $\dR = \sqrt{\deta^2 + \dphi^2}$ is less than a certain threshold. 
-For the photon candidate object, a line that connects the detector origin and the cluster position was used to define its direction because photons leave no tracks and do not bend in the magnetic field.
+A trigger object is matched to the candidate when their angular separation  $\dR = \sqrt{\deta^2 + \dphi^2}$ is less than a certain threshold. \PH{Wow sudeenly here $\DeltaR$ is defined. this should have come earlier}
```

Comment:
dR is defined way back at the beginning of the detector chapter. this is just a reminder for convenience.